src/lib.rs

```diff
@@ -51,6 +55,9 @@ pub struct Contract {
     pub label_to_proposals: UnorderedMap<String, HashSet<ProposalId>>,
     pub author_proposals: UnorderedMap<AccountId, HashSet<ProposalId>>,
     pub proposal_categories: Vec<String>,
+    pub rfps: Vector<VersionedRFP>,
+    pub label_to_rfps: UnorderedMap<String, HashSet<RFPId>>,
+    pub global_labels_info: UnorderedMap<String, LabelInfo>,
```

Comment:
There won't be more than 50 items there, so it is cheaper to store (serialize & deserialize) the global labels in a single storage value than to introduce individual key-value record per label.

I suggest you use [`near_sdk::store::Lazy<std::collections::HashMap<String, LabelInfo>>`](https://docs.rs/near-sdk/latest/near_sdk/store/struct.Lazy.html), so we don't get penalized for deserializing HashMap on every function call